roles/sap_netweaver_preconfigure/tasks/sapnote/assert-3119751.yml

```diff
@@ -5,46 +5,77 @@
     msg: "SAP note {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).number }}
           (version {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).version }}): Linux Requirements for SAP Kernel 754 and for SAP Kernel 788 and higher"
 
-- name: Get info about the compat-sap-c++-10.so file
+# Note: This file is only included for RHEL 8, so no further when condition is required here.
+
+- name: Identify the latest compat-sap-c++-NUM.so file with NUM >= 10 # noqa risky-shell-pipe
+  ansible.builtin.shell:
+  args:
+    cmd: ls compat-sap-c++-1* | sort | tail -1
```

Comment:
Same here, I suggest using the `file` module instead of shell.

Reply:
See above, now fixed.